test/Spec.hs

```diff
@@ -631,4 +640,51 @@ getAttachmentsFromCommentSpec =
             all (\att -> aContentType att `elem` zipFileContentTypes) filteredAttachments
           where
             zipFileContentTypes :: [Text]
-            zipFileContentTypes = ["application/zip", "application/x-zip-compressed"]
\ No newline at end of file
+            zipFileContentTypes = ["application/zip", "application/x-zip-compressed"]
+
+------------------------------------------------------------
+-- Tests for database operations
+------------------------------------------------------------
+
+deleteAllDataSpec :: Spec
```

Comment:
Good work. Will take a more detailed look when you think you covered the exceptional situations that can arise from the database.

Reply:
Yes, I'll try to add more test cases. I think I can use mockery to create temporary database with certain conditions!